data/lang/module-collectdata/en.json

```diff
@@ -0,0 +1,74 @@
+{
+  "DATACOLLECTION": {
+    "description": "",
+    "message": "Data collection"
+  },
+  "THANKS_I_CANT_WAIT_FOR_THE_RESULTS": {
+    "description": "",
+    "message": "Thanks, we can't wait for the results!"
+  },
+  "OK_AGREED": {
+    "description": "",
+    "message": "Ok, agreed."
+  },
+  "INTERGALACTIC_RESEARCH_FOUNDATION": {
+    "description": "",
+    "message": "Intergalactic Research Foundation"
+  },
+  "EXTRATERESTIAL_SCIENCE_LABS": {
+    "description": "",
+    "message": "Extraterestial Science Labs"
+  },
+  "EARTH_FEDERATION_OF_SCIENCE": {
+    "description": "",
+    "message": "Earth Federation of Science"
+  },
+  "INDEPENDENT_SCIENTISTS": {
+    "description": "",
+    "message": "Independent Scientists"
+  },
+  "VERY_GOOD_I_WILL_SEND_THIS_DATA_DIRECTLY_TO_THE": {
+    "description": "",
+    "message": "Very good, I will send this data directly to the "
+  },
+  "NOT_LOSE_ENOUGH_TO": {
+    "description": "",
+    "message": "Not close enough to "
+  },
+  "TO_COLLECT_DATA": {
+    "description": "",
+    "message": " to collect data."
+  },
+  "DATA_FROM": {
+    "description": "",
+    "message": "Data from "
+  },
+  "COLLECTED": {
+    "description": "",
+    "message": " collected"
+  },
+  "SET_TO_BE_INSPECTED_PLANET_AS_TARGET": {
+    "description": "",
+    "message": "Set to be inspected planet as target"
```

Comment:
Took a few times reading this to figure out what it means. Hyphens are important. "Set to-be-inspected planet as target" is understood easily. And for the next entry, perhaps "Set delivery station as target" as the "station to bring the data to" is a bit awkward.

Reply:
Or "_Set planet as target to be inspected_" for a sentence that makes a bit more sense :D

Reply:
Is the current standard "Set destination as navigation target" not accurate for this mission type?

Otherwise, for consistency: "Set planet to be inspected as navigation target"?

Is the destination always a planet? The initial description suggests not... Perhaps change the word "planet" to "body"?